qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -310,29 +337,6 @@ def __init__(self, name, setup_folder, address, reset=False,
                     vals=vals.Numbers(-2.7, 2.7),
                     get_parser=float)
 
-        # # Add functions
```

Comment:
@AdriaanRol  commented code was removed, and in many cases the functions where not even defined.  So in all effects this particular change is  a NOP 🎉 

Functions are still there as methods/parameters or as qcodes.functions where it makes sense to @WilliamHPNielsen (and also according to the documentation of the Function class),  see line 143.
I would say that many of the comment lines were pretty good candiate for being parameters.
example
```python 
# self.add_function('set_event_jump_timing')
# self.add_function('get_event_jump_timing')
```
became: 
```python
        self.add_parameter('event_jump_timing',
                           get_cmd='EVENt:JTIMing?',
                           set_cmd='EVENt:JTIMing {}',
                           vals=vals.Enum('SYNC', 'ASYNC'))
 ```
Hope this explains !